app/api_alpha/urls.py

```diff
@@ -34,6 +35,7 @@
     path("buildings/closest/", BuildingClosestView.as_view()),
     path("", include(router.urls)),
     path("login/", auth_views.obtain_auth_token),
-    path("tiles/<int:x>/<int:y>/<int:z>.pbf", get_tile),
+    path("tiles/<int:x>/<int:y>/<int:z>.pbf", get_tile_point),
```

Comment:
Qu'est ce que tu penses d'introduire une nouvelle adresse de type "points" pour nos tuiles actuelles pour uniformiser avec "shapes" ? On garderait l'ancienne adresse également l'ancienne mais sans la diffuser dans notre documentation.

```suggestion
    # We keep this url for legacy purpose
    path("tiles/<int:x>/<int:y>/<int:z>.pbf", get_tile_point),
    path("tiles/points/<int:x>/<int:y>/<int:z>.pbf", get_tile_point),
```